### PR TITLE
Feature/viewbinding and sdk 34 migration

### PR DESCRIPTION
**Pull request by fatihergin**

### CODE REVIEW

**Comment by tibbi** [comment]
settings page doesnt have transparent navigation, check all apps please

File: app/src/main/kotlin/com/simplemobiletools/keyboard/activities/MainActivity.kt [hunk 1]
```diff
@@ -12,28 +12,32 @@ import com.simplemobiletools.commons.helpers.LICENSE_GSON
 import com.simplemobiletools.commons.models.FAQItem
 import com.simplemobiletools.keyboard.BuildConfig
 import com.simplemobiletools.keyboard.R
-import kotlinx.android.synthetic.main.activity_main.*
+import com.simplemobiletools.keyboard.databinding.ActivityMainBinding
 
 class MainActivity : SimpleActivity() {
+    private val binding by lazy(LazyThreadSafetyMode.NONE) { ActivityMainBinding.inflate(layoutInflater) }
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/keyboard/activities/MainActivity.kt:18]: lets actually use `viewBinding` like in Dialer, that is the helper extension function

Comment (fatihergin) [reply app/src/main/kotlin/com/simplemobiletools/keyboard/activities/MainActivity.kt]: done

**Comment by fatihergin** [comment]
> Settings page doesnt have transparent navigation, check all apps you rewrote please. Even "Manage clipboard items" works well.

it's already `false` on the [current master](https://github.com/SimpleMobileTools/Simple-Keyboard/blob/2c8edfac94737954307461fd0fb5f88689fa37f9/app/src/main/kotlin/com/simplemobiletools/keyboard/activities/SettingsActivity.kt#L26). didn't touch it @tibbi

**Comment by tibbi** [comment]
hm ok, not sure why was it like that. Looks good though, thanks

**Pull request merged**